compiler/fir/analysis-tests/testData/resolve/diagnostics/functionAsExpression/Parameters.kt

```diff
@@ -0,0 +1,16 @@
+val bar = fun(<!ANONYMOUS_FUNCTION_PARAMETER_WITH_DEFAULT_VALUE{LT}!>p: Int = <!ANONYMOUS_FUNCTION_PARAMETER_WITH_DEFAULT_VALUE{PSI}!>3<!><!>) {}
```

Comment:
As mentioned https://github.com/JetBrains/kotlin/pull/4036#discussion_r559333938, this is the place I first noticed LT positioning strategies may not be working under the current setting. I can't even set a break point and debug new strategies' `mark` overrides.

At least, PSI-based tree and diagnostic results match with old FE.

Reply:
Fixed at https://github.com/JetBrains/kotlin/commit/40bec30393dea6462ba2548da88296a80d99f15a woohoo~